couch/metadata.csv

```diff
@@ -261,6 +261,11 @@ couchdb.erlang.memory.other,gauge,,byte,,,0,couchdb,,
 couchdb.erlang.memory.processes,gauge,,byte,,,0,couchdb,,
 couchdb.erlang.memory.processes_used,gauge,,byte,,,0,couchdb,,
 couchdb.erlang.message_queues.size,gauge,,,,,0,couchdb,,
+couchdb.erlang.message_queues.max,gauge,,,,,0,couchdb,,
+couchdb.erlang.message_queues.min,gauge,,,,,0,couchdb,,
+couchdb.erlang.message_queues.50,gauge,,,,,0,couchdb,,
+couchdb.erlang.message_queues.90,gauge,,,,,0,couchdb,,
+couchdb.erlang.message_queues.99,gauge,,,,,0,couchdb,,
```

Comment:
Non blocking nit: can you also leave a brief description describing these metrics?